### ISSUE TITLE
Fix nullable reference types warning

### ISSUE THREAD

**Issue by nickiemc** [corgibytes/freshli#199]
When building, the following warning is produced:

```
Languages/Python/Conversions.cs(29,74): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context. 
```

This warning should be addressed.

**Comment by mrbiggred** [comment]
@dan-hein I noticed that @nickiemc re-opened this case.  Can you take a look to see if PR #213 actually fixed this issue or not?  Thanks.

**Comment by dan-hein** [comment]
Hey @mrbiggred, I believe Nickie did that just to mark the task as Ready for Deployment. The changes attached should fix the issue described. I think that was mainly for the Freshli board.

**Comment by nickiemc** [comment]
@dan-hein @mrbiggred Correct. GitHub automatically closed the issue because of the `Resolves` keyword in PR #213. But since it still hasn't been deployed to production, I reopened it and moved to Ready for Deployment.
